.rubocop.yml

```diff
@@ -1,12 +1,28 @@
+AllCops:
+  TargetRubyVersion: 2.3
+  Exclude:
+    - 'bin/**/*'
+    - 'db/**/*'
+    - 'config/**/*'
+    - 'spec/**/*'
+
 Documentation:
   Enabled: false
 
+Rails:
+  Enabled: true
+
+# There is a bug in the Rubocop 0.48 parser that causes it to incorrectly
+# complain about parenthese around blocks used with scopes. This should be
+# removed once it is possible to upgrade to 0.49 or above.
+Lint/AmbiguousBlockAssociation:
+  Enabled: false
+
 Style/FrozenStringLiteralComment:
   Enabled: false
 
-AllCops:
-  Exclude:
-    - 'db/**/*'
-    - 'config/**/*'
-    - 'spec/**/*'
-    - !ruby/regexp /old_and_unused\.rb$/
+# Rubocop should be used for style guide adherence, not testing for code smell
+Rails/OutputSafety:
+  Enabled: false
+Rails/SkipsModelValidations:
+  Enabled: false
```

Comment:
The [RuboCop documentation](http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Rails/SkipsModelValidations) states that this "checks for the use of methods which skip validations".
I'll 100% support your decision—and this is a completely subjective matter—but my opinion is that this is RuboCop checking for code smell.

We can wrap these calls in "rubocop:disable" statements if you choose to continue throwing errors about this.

I like using reek for code smell and RuboCop for enforcing a style guide. I'm sure there would be others with a different opinion. Either way, I'll follow your guidance on the matter since it is so subjective.